coordinator/src/routes/util.ts

```diff
@@ -39,6 +39,10 @@ export function handleResponse<T>(
     return responseToolkit.response(spineResponse.body)
       .code(spineResponse.statusCode)
       .header("Content-Type", contentType)
+  } else if (isBundle(spineResponse.body)) {
```

Comment:
when would spine respond with a bundle?

Reply:
It wouldn't, but the sandbox client does (for now). This can probably be removed once we've sorted the sandbox interactions for dispensing.

Reply:
without the `isBundle` check it would fall through to `translateToFhir` right? and release responses are translated there by your translations